src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/AltSiteRecord.java

```diff
@@ -0,0 +1,125 @@
+package org.broadinstitute.hellbender.tools.walkers.readorientation;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Nucleotide;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by tsato on 10/11/17.
+ */
+public class AltSiteRecord {
```

Comment:
If you imagine that only the ref and the main alt will be important to your model, you could consider unifying thi with `PileupSummary`.  It might *not* make sense, but think about it a bit.

Reply:
I do feel that keeping counts for all 4 bases might be overkill, but I also felt weird about pegging other allele counts to 0. If after more testing I find that I only need ref and main alt, I'll refactor

Reply:
How would you unify with PileupSummary, though? The columns are similar but they're different

Reply:
You would expand it to include some columns needed for orientation bias but not contamination and vice versa.  There's little harm because I/O of the bam is by far the dominant factor in run time of these tools.

Reply:
I chewed on this a bit more. I think CalculateContamination and orientation bias sharing a table is not ideal. The real issue here seems to be that creating a new tsv writer/reader e.g. AltSiteRecord requires so much code writing, and most of the code is copied and pasted from a similar class. I think we can trade off some flexibility in `TableWriter` and `TableReader` for convenience.

...but that's not high priority right now, and I will keep AltSiteRecord for the time being since it's already written.

Reply:
That's fine.  Some time down the line we will probably coordinate with the CNV team to consolidate several related pileup-related classes.